climada/engine/impact.py

```diff
@@ -510,6 +514,79 @@ def write_sparse_csr(self, file_name):
         np.savez(file_name, data=self.imp_mat.data, indices=self.imp_mat.indices,
                  indptr=self.imp_mat.indptr, shape=self.imp_mat.shape)
 
+    def write_netcdf(self, path=None, attrs=None, with_imp_mat=True, **kwargs):
```

Comment:
To keep things simple I suggest to not use argument forwarding unless they always can be forwarded as a whole and untouched:
` rasterize(**kwargs)`
Otherwise one does not readily see what arguments are used.
In other words, I'd prefer to have the two additional arguments `event_id` and `drop_zero` explicit in the signature.